src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -194,9 +195,13 @@ else if (!StringUtils.isNullOrEmpty(setup))
         {
             dtlsStack.setRemoteFingerprints(remoteFingerprints);
 
-            String hash = remoteFingerprints.entrySet().iterator().next().getKey();
+            final boolean hasSha1Hash = remoteFingerprints
+                .keySet()
+                .stream()
+                .anyMatch(hash -> hash.equalsIgnoreCase("sha1"));
```

Comment:
It used to check for 'sha-1' string and now this is changed to 'sha1', is this intentional?

Reply:
Thanks a lot @damencho, nice catch! It was completely accidental change.